### DISCUSSION TITLE
How do you serve multiple edge-functions locally?

### DISCUSSION THREAD

**Discussion by ryanravn** [supabase/supabase#7845] [Questions] [answered]
I'm running the local developer suite with Supabase CLI. Whenever I try to serve a new function with` supabase functions serve [name]`, the old one stops, is there a way to bypass this?

**Comment by rustamd** [comment]
Haven't tried it, but maybe just `supabase functions serve`?

**Comment by GaryAustin1** [comment, marked as answer]
No.  This is current limitation.
https://supabase.com/docs/guides/functions#limitations
Although there is this suggestion from a user: https://github.com/supabase/supabase/discussions/6786

**Reply by soedirgo** [reply]
This is currently tracked on https://github.com/supabase/cli/issues/459

**Comment by SteveChurch** [comment]
It would be nice to see a long term solution for this and maybe more push for it. I'm not sure how I can possibly run my project locally without having to have all of my edge functions running. I might try the suggested #6786
